Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -43,6 +43,17 @@ struct TabsButtonUX {
 }
 
 class TabsButton: UIControl {
+    var previousTheme = Theme.NormalMode
```

Comment:
`previousTheme` -> `theme`. You're updating the theme whenever it's applied, meaning it's the active theme, right?